presentation/src/main/java/com/keeply/presentation/ui/onboarding/OnboardingScreen.kt

```diff
@@ -0,0 +1,161 @@
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.graphics.Brush
+import androidx.compose.ui.graphics.Color
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.hilt.navigation.compose.hiltViewModel
+import com.kakao.sdk.auth.model.OAuthToken
+import com.kakao.sdk.common.model.ClientError
+import com.kakao.sdk.common.model.ClientErrorCause
+import com.kakao.sdk.user.UserApiClient
+import com.keeply.presentation.R
+import com.keeply.presentation.core.components.KeeplyButton
+import com.keeply.presentation.core.theme.KeeplyTheme
+import com.keeply.presentation.ui.onboarding.component.KakaoLoginButton
+import com.keeply.presentation.ui.onboarding.component.OnboardingPager
+import org.orbitmvi.orbit.compose.collectAsState
+
+@Composable
+fun OnboardingRoute(
+    viewModel: OnboardingViewModel = hiltViewModel()
+) {
+    val uiState by viewModel.collectAsState()
+
+    OnboardingScreen(
+        onboardingPage = uiState.onboardingPage,
+        updateOnboardingPage = viewModel::updateOnboardingPage
+    )
+}
+
+@Composable
+fun OnboardingScreen(
+    onboardingPage: OnboardingPage,
+    updateOnboardingPage: (OnboardingPage) -> Unit = {}
+) {
+    val context = LocalContext.current
+
+    Box(
+        modifier = Modifier
+            .fillMaxSize()
+            .background(KeeplyTheme.colors.neutral100)
+    ) {
+        OnboardingPager(
+            modifier = Modifier
+                .padding(top = 40.dp)
+                .width(246.dp)
+                .align(Alignment.TopCenter),
+            onboardingPage = onboardingPage
+        )
+
+        Box(
+            modifier = Modifier
+                .fillMaxWidth()
+                .height(270.dp)
+                .align(Alignment.BottomCenter)
+                .background(
+                    brush = Brush.verticalGradient(
+                        colors = listOf(
+                            Color.Transparent,
+                            KeeplyTheme.colors.neutral100,
+                            KeeplyTheme.colors.neutral100
+                        )
+                    )
+                )
+        )
+
+        val bottomModifier = Modifier
+            .padding(
+                start = 16.dp,
+                end = 16.dp,
+                bottom = 50.dp
+            )
+            .fillMaxWidth()
+            .align(Alignment.BottomCenter)
```

Comment:
p1: button height 추가부탁드립니다~

```suggestion
        val bottomModifier = Modifier
            .padding(
                start = 16.dp,
                end = 16.dp,
                bottom = 50.dp
            )
            .fillMaxWidth()
            .height(50.dp)
            .align(Alignment.BottomCenter)
```

Reply:
수정 반영했어요~

Reply:
감사합네다~